types/timepicker/index.d.ts

```diff
@@ -0,0 +1,164 @@
+// Type definitions for Timepicker Plugin for jQuery 1.13
+// Project: <https://github.com/jonthornton/jquery-timepicker>
+// Definitions by: doberkofler <https://github.com/doberkofler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.0
+
+/// <reference types="jquery"/>
+
+/* tslint:disable:unified-signatures */
```

Comment:
This needs to be disabled only for the lines it affects and not for the whole file.

Reply:
I use eslint and have never used tslint. How can I disable a single line?

Reply:
```ts
// tslint:disable-line:unified-signatures
```

or 
```ts
// tslint:disable-next-line:unified-signatures
```